ajax/libs/markerclustererplus/2.0.15/markerclusterer.min.js

```diff
@@ -0,0 +1,2 @@
+//# sourceMappingURL=markerclusterer.min.js.map
```

Comment:
@pvnr0082t How to minify files using the `web-minify-helper/minify.sh` without creating the `.map` file?